### ISSUE TITLE
Unable to connect to IP camera Internec i6.4-C series

### ISSUE THREAD

**Issue by kfam-marcink** [motioneye-project/motioneye#2868]
Hello,
I'm trying to connect to the rtsp stream from camera as in the topic but in the field camera nothing apperars. I'm connecting from HAOS add-on. 
![image](https://github.com/motioneye-project/motioneye/assets/63196794/909baea5-c69a-47b3-b97d-549303135f41)
As I try to connect with the same link with VLC I have no problem getting to the video stream.
What am I doing wrong?
The link I'm using: rtsp://user:password@1xx.1xx.10.1xx/1/2/

**Comment by acyanic** [comment]
just passing by, so my guess..
you have username and password in the URL as well as the boxes.
Try one (probably just in the boxes), not both.

**Comment by geolr** [comment]
I think the url needs review, due to the @ it's not validated (red icon to the right in the textfield).

Try without `user:password@` in the url like:
rtsp://192.168.n.n/...

**Comment by MichaIng** [comment]
Yes, URLs with embedded credentials are not even supported by modern browsers when present in the HTML document. In this case, when it is processed in the backend, this might not be the blocker, but we have the input fields to send them via HTTP authentication, which should be used (only) instead.
